edge-hub/watchdog/linux/Makefile

```diff
@@ -0,0 +1,13 @@
+TARGET=target/release
```

Comment:
just curious why do we need makefile?

Reply:
There is a script in the build pipeline that will cross compile rust artifacts. It is modeled after the build process for edgelet, which calls make in a certain directory. I chose to keep it consistent.